custom_components/powercalc/data/yeelight/YLDD04YL/2_meter_extension/model.json

```diff
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plug S",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.4.0:docker"
+    },
+    "name": "Yeelight Lightstrip Plus with extension",
```

Comment:
All these properties won't be used for sub profiles.
You'll need to put the model.json in the root (directory above).
Only separate model.json file is needed when standby_power differentiates per sub profile.
But that is not the case for you. 0.53 and 0.54 is just some minor measurement variation.